serving/accessing-metrics.md

```diff
@@ -33,8 +38,42 @@ The following dashboards are pre-installed with Knative Serving:
 4. Set up an administrator account to modify or add dashboards by signing in
    with username: `admin` and password: `admin`.
 
-- Before you expose the Grafana UI outside the cluster, make sure to change the
-  password.
+  - Before you expose the Grafana UI outside the cluster, make sure to change the
+  password. You will be prompted to set a password on first login, and it can
+  later be changed at
+  [http://localhost:3000/org/users](http://localhost:3000/org/users).
+
+
+## Prometheus
+
+Grafana provides a richer set of graphing functionality than Prometheus.
+However, it can often times be useful to access Prometheus for debugging.
```

Comment:
"Grafana provides a richer set of graphing functionality than Prometheus.
However, it can often times be useful to access Prometheus for debugging."
--->
"Although Grafana provides a richer set of graphing functionality, Prometheus can still be useful for debugging."